mesonbuild/dependencies/ui.py

```diff
@@ -187,13 +187,11 @@ class VulkanDependencySystem(SystemDependency):
     def __init__(self, name: str, environment: 'Environment', kwargs: T.Dict[str, T.Any], language: T.Optional[str] = None) -> None:
         super().__init__(name, environment, kwargs, language=language)
 
-        try:
-            self.vulkan_sdk = os.environ.get('VULKAN_SDK', os.environ['VK_SDK_PATH'])
-            if not os.path.isabs(self.vulkan_sdk):
-                raise DependencyException('VULKAN_SDK must be an absolute path.')
-        except KeyError:
-            self.vulkan_sdk = None
+        self.vulkan_sdk = os.environ.get('VULKAN_SDK', None) or os.environ.get('VK_SDK_PATH', None)
+        if self.vulkan_sdk and not os.path.isabs(self.vulkan_sdk):
+            raise DependencyException('VULKAN_SDK must be an absolute path.')
```

Comment:
I don't really understand the changes you're making here. This should *just* be:

```diff
-            self.vulkan_sdk = os.environ.get('VULKAN_SDK', os.environ['VK_SDK_PATH'])
+            self.vulkan_sdk = os.environ.get('VULKAN_SDK', os.environ.get('VK_SDK_PATH'))

Reply:
Thanks @eli-schwartz. I was overthinking it a bit.